### PR TITLE
Move self-label button

### PR DESCRIPTION
**Pull request by haileyok**
Screenshot 2024-10-16 at 3 58 32 PM

Screenshot 2024-10-16 at 3 58 41 PM

### CODE REVIEW

File: src/view/com/composer/Composer.tsx [hunk 1]
```diff
@@ -758,22 +744,37 @@ export const ComposePost = ({
...
-                  type: 'update_threadgate',
-                  threadgate: nextThreadgate,
-                })
-              }}
-              style={bottomBarAnimatedStyle}
-            />
-          )}
+          <Animated.View
+            style={[a.flex_row, a.p_sm, t.atoms.bg, bottomBarAnimatedStyle]}>
+            <ScrollView
```
Comment (haileyok) [inline src/view/com/composer/Composer.tsx:749]: A `ScrollView` because we want to handle accessibility font sizes gracefully.

**Comment by surfdude29** [comment]
This looks great, thanks @haileyok! :)

**Comment by mary-ext** [comment]
`Self-labels` seems non-descriptive, but I understand that there is not a lot of horizontal space for this, could we go for something like `Add labels` or `Add warning`?

**Comment by haileyok** [comment]
Yea seems legit, I had some hesitation around that wording as well. Sorta think we could rephrase the interactions button too.

**Review by mozzius — APPROVED** [review]
lgtm. I put my suggestions in a PR, but this is good

**Pull request merged**